.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+on:
+  push:
+    tags:        
+      - 'v*'
+jobs:
+  test:
+    uses: ./.github/workflows/test.yml
+  npm-publish:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run build
```

Comment:
Apparently `npm ci` already does this. I suppose doing it again can't hurt, so the question is more "is it more important to futureproof against changes where `npm ci` no longer builds, or more important to have the release workflow accurately reflect what steps do what."

Reply:
True. I was looking inside the build script and did see that it also installed dependencies. My main reason for doing it here was just in case that script ever changed, as _in theory_ all a build script should do is build, not install dependencies

I don't think there's any harm in running the dependency install twice. `npm ci` is the same as `npm i` if I recall correctly, but it just fails if the package.json and package-lock.json don't match, as an extra layer of protection

Reply:
We seem to have a postinstall script that builds.

Reply:
Oh huh, I didn't see that script... I suppose it does. I'm a little confused what the best practice here is then. Installing dependencies builds the app, but also building the app installs dependencies

We could keep things simple and just only run `npm ci`, but if that postinstall script is ever removed it'd break things

In theory there's no harm in running both the install and build steps in CI, so perhaps for clarity we should leave them both in?

Reply:
I don't have a strong opinion here. It seems you prefer to leave both in, which is fine by me.